uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java

```diff
@@ -350,13 +350,34 @@ public void setFlights(@Nullable final String flightsJson) {
         }
 
         // scroll to find the set flights button
-        UiAutomatorUtils.obtainChildInScrollable("Set Flights");
+        UiAutomatorUtils.obtainChildInScrollable("Set flights (When BrokerHost is the active broker)");
         // input flights string in flights input box
         UiAutomatorUtils.handleInput("com.microsoft.identity.testuserapp:id/editTextFlights", flightsJson);
         // Click Set Flights button
         UiAutomatorUtils.handleButtonClick("com.microsoft.identity.testuserapp:id/setFlightsButton");
     }
 
+    @Override
+    public void addFlights(@Nullable final String flightsJson) {
+        Logger.i(TAG, "Add Flights..");
+        launch();
+
+        // Sleep for a bit to finish launching brokerHost before scrolling to set Flights
+        try {
+            Thread.sleep(TimeUnit.SECONDS.toMillis(5));
```

Comment:
is it possible to add this delay to "obtainChildInScrollable" to wait till it finds the element.

Reply:
obtainChildInScrollable was also used by other tests.